frontend/src/metabase/visualizations/lib/save-dashboard-pdf.ts

```diff
@@ -217,11 +217,13 @@ export const saveDashboardPdf = async ({
     headerHeight + parametersHeight + (includeBranding ? brandingHeight : 0);
   const contentHeight = gridNode.offsetHeight + verticalOffset;
 
-  let backgroundColor = Color(
-    getComputedStyle(document.documentElement)
-      .getPropertyValue("--mb-color-bg-dashboard")
-      .trim(),
-  ).hex();
+  const rawBackgroundColor = getComputedStyle(document.documentElement)
+    .getPropertyValue("--mb-color-bg-dashboard")
+    .trim();
+  let backgroundColor =
+    rawBackgroundColor === "transparent"
+      ? "transparent"
+      : Color(rawBackgroundColor).hex();
 
   if (!(await isValidColor(backgroundColor))) {
     backgroundColor = "white"; // Fallback to white if the color is invalid
```

Comment:
maybe it'd be good to fallback with whatever comes out of Color.hex() here, for consistency's sake?